.github/pull_request_template.md

```diff
@@ -0,0 +1,25 @@
+<!--
+Thank you for sending the PR! We appreciate you spending the time to work on these changes.
+
+Help us understand your motivation by explaining why you decided to make this change.
+You can learn more about contributing to Mypy here: https://github.com/python/mypy/blob/master/CONTRIBUTING.md
+
+Happy Contributing!
+
+-->
+
+### Have you read the [Contributing Guidelines](https://github.com/python/mypy/blob/master/CONTRIBUTING.md)?
+
+(Write your answer here.)
+
+### Description
+
+(Write your answer here.)
```

Comment:
```suggestion
(Explain how this PR changes mypy.)
```
And honestly if there are any related issues or PRs I'd like to see them listed here, not in a separate section at the bottom. Ideally in the form "Closes #NNN" or "Fixes #NNN" (we should explicitly mention these, many people don't know about those!)